### ISSUE TITLE
Test suite failed to run

### ISSUE THREAD

**Issue by gjolund** [ricglz0201/pokemon-relay-app#2]
Seems like the jest config isn't running the babel relay compiler.

```sh
Invariant Violation: graphql: Unexpected invocation at runtime. Either the Babel transform was not set up, or it failed to identify this call site. Make sure it is being used verbatim as `graphql`.

      20 |   PokemonDetails,
      21 |   {
    > 22 |     pokemon: graphql`
         |                      ^
      23 |       fragment PokemonDetails_pokemon on Pokemon {
      24 |         id
      25 |         name

      at invariant (node_modules/fbjs/lib/invariant.js:40:15)
      at graphql (node_modules/relay-runtime/lib/query/GraphQLTag.js:24:52)
      at Object.<anonymous> (src/PokemonDetails.tsx:22:22)
      at Object.<anonymous> (src/PokemonList.tsx:1:1)
      at Object.<anonymous> (src/App.tsx:1:1)
      at Object.<anonymous> (src/App.test.tsx:3:1)
      at processTicksAndRejections (node:internal/process/task_queues:93:5)
```

**Comment by ricglz** [comment]
Thanks for pointing this out! I will check it, the weird thing is that seems to be only for running the tests

**Comment by ricglz** [comment]
Solved in #3

**Comment by gjolund** [comment]
thanks @ricglz0201

nice to see that it dropped the rewired dependency too!
